### ISSUE TITLE
Web UI: Move and clarify informed consent to better match the join flow

### ISSUE THREAD

**Issue by NickAraujo** [google/paco#1358]
The Informed Consent info should be moved to the Basics tab, before "Instructions," because this is when it is actually presented to the user.

We might want to include the boiler plate text so that researchers have a better understanding of what else will be presented on-screen in addition to their custom informed consent info:

# Informed Consent
## You Agree to Share Information with the Experimenters
By joining this experiment, you will be sharing your responses with the creator and administrators of this experiment, and they may use that information for research, product development, or other purposes. Please read the privacy policy or data handling statement they have provided below to decide whether you want to participate in this experiment.

## Types of Data Collected
Check all that apply. Some data types will automatically be checked for you as you build your experiment. (Note: Is this possible?)
* User responses to experiment form (Note: is there ever a case where this won't be true?--i.e., should this be a checkbox that is automatically checked if they add an input question?)
* App usage and browser history
* Location information
* Apps installed on your phone
* Details about your phone (make, model, carrier)

## Experimenter's Informed Consent Statement
(Note: Should this be "privacy policy" or "data handling statement" to match with the text above--or the text changed to match with this title?)

**Comment by BobEvans** [comment]
I agree with this. Also, there is already a bug filed for this. Please merge the two and mark one as a duplicate and close it.

**Comment by ianspiro** [comment]
Any more thoughts on what the boilerplate consent text should be?

**Comment by NickAraujo** [comment]
On second thought, let's leave the boilerplate text in an external help doc, since it may vary a lot by institution.

Expanding the scope a bit here:
1. Move the phone details, informed consent, and extra types of data collection to the Basics tab.
2. Rename some of these fields to be more intuitive.

Here's the new order/names:
* ...other items on the Basics tab...
* Ringtone
* (checkbox) Record phone details on join (help link: Using Advanced Data Collection Features)
* Experimenter's Informed Consent Statement (help link: Informed Consent)
* Advanced: Disclose Data Collected by Custom Scripts (help link: Using Advanced Data Collection Features)

3. Link "Log app usage and browser history" on the Groups tab to the appropriate data declaration, so that it appears on the app if a researcher selects this option.

Eventually we will consider programmatically hiding the advanced data declarations and reconsider the grouping/names for the tabs (will be a separate bug).
